src/esp/bindings/bindings.cpp

```diff
@@ -606,13 +578,14 @@ PYBIND11_MODULE(habitat_sim_bindings, m) {
   py::class_<SimulatorConfiguration, SimulatorConfiguration::ptr>(
       m, "SimulatorConfiguration")
       .def(py::init(&SimulatorConfiguration::create<>))
-      .def_readwrite("agents", &SimulatorConfiguration::agents)
       .def_readwrite("scene", &SimulatorConfiguration::scene)
       .def_readwrite("default_agent_id",
                      &SimulatorConfiguration::defaultAgentId)
       .def_readwrite("default_camera_uuid",
                      &SimulatorConfiguration::defaultCameraUuid)
       .def_readwrite("gpu_device_id", &SimulatorConfiguration::gpuDeviceId)
+      .def_readwrite("width", &SimulatorConfiguration::width)
```

Comment:
Do we need to add `width` and `height` as sim-level properties?  Introducing these is a bit counter to supporting multi-sensor setups with differing resolutions and might be confusing longer-term

Reply:
Currently these are needed as the renderer owns the global frame buffer.  The current code just looks up the width and height of sensor 0 and uses that for all the rendering.